streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregateProcessorTest.java

```diff
@@ -345,8 +350,49 @@ public void shouldLogAndMeterWhenSkippingLateRecord() {
                 mkEntry("task-id", "0_0"),
                 mkEntry("processor-node-id", "TESTING_NODE")
             )
-        ));
-        assertEquals(1.0, dropMetric.metricValue());
+        );
+
+        assertThat(metrics.metrics().get(dropMetric).metricValue(), is(2.0));
+
+        final MetricName dropRate = new MetricName(
+            "late-record-drop-rate",
+            "stream-processor-node-metrics",
+            "The average number of occurrence of late-record-drop operations.",
+            mkMap(
+                mkEntry("client-id", "test"),
+                mkEntry("task-id", "0_0"),
+                mkEntry("processor-node-id", "TESTING_NODE")
+            )
+        );
+
+        assertThat(metrics.metrics().get(dropRate).metricValue(), not(0.0));
```

Comment:
The actual value depends on how quickly the test runs, but we do know it shouldn't be 0, which is sufficient to demonstrate that *something* got recorded.

Reply:
super nit: should we assert greater than `0.0`? But I don't have a strong opinion here though.

Reply:
I was being lazy, since we're just coincidentally pulling in some, but not all, of hamcrest, we don't have access to the "greater than" matcher.

I've added the dependency now, though, so I can add this matcher as you suggest.